validator/api/src/test/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocatorTest.java

```diff
@@ -103,6 +104,27 @@ public void shouldIgnoreSomeFiles(@TempDir final Path tempDir) throws IOExceptio
                 tempDir, Path.of("key", "a.json").toString(), Path.of("pass", "a.txt").toString()));
   }
 
+  @Test
+  public void shouldIgnoreDepositDataJsonFile(@TempDir final Path tempDir) throws IOException {
+    createFolders(tempDir, "key", "pass");
+    createFiles(
+        tempDir,
+        Path.of("key", "deposit-data.json"),
+        Path.of("key", "a.json"),
+        Path.of("pass", "deposit-data.txt"),
```

Comment:
The test should pass without this `deposit-data.txt` - there won't normally be a password file for the deposit data file.